script/toolforge-cron.daily

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-set -e
```

Comment:
Commit message says we're removing `set -x`.

Reply:
🤦‍♂️

Reply:
Fixed and force pushed.